src/tests/createDefaultCodec.test.ts

```diff
@@ -38,6 +38,12 @@ describe('buildDefaultCodec', () => {
     expect(defaultCodec).to.equal('A');
   });
 
+  it('should correctly build a default codec for an object', () => {
+    const codec = t.object;
+    const defaultCodec = createDefaultCodec(codec);
+    expect(defaultCodec).to.deep.equal({});
```

Comment:
Does this also enforce that there are no keys in the resulting object?

Reply:
Yes! The test fails if the object has any keys.